server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -67,6 +71,8 @@ export type DraftAppointmentBooking = null | AppointmentSchedulingDetails
 export default class ServiceProviderReferralsController {
   private readonly deliusOfficeLocationFilter: DeliusOfficeLocationFilter
 
+  private paginationFlow = true
```

Comment:
let's ship this first with the old non-paged version and turn it on in a later commit